framework/src/utils/FormattedTable.C

```diff
@@ -247,7 +247,21 @@ FormattedTable::printTable(std::ostream & out,
   else if (suggested_term_width == "AUTO")
     term_width = getTermWidth(false);
   else
-    term_width = suggested_term_width;
+  {
+    try
+    {
+      // convert the name into a number, without this conversion the id is used which
+      // is assigned automatically and is not related to the name
+      term_width = std::stoi(suggested_term_width);
```

Comment:
I'm pretty sure this isn't good enough. @rwcarlsen - Didn't you run into this same problem very recently? This will still partially convert a number that's part of a string. We've solved this in other places in the framework. Take a look at all the boundary and block conversion logic in MooseMesh for instance.

Reply:
You are right, the following results in "num = 200".

```
#include <iostream>
#include <string>
int main ()
{
  std::string foo = "200letters";
  int num = std::stoi(foo);
  std::cout << "num = " << num << std::endl;
  return 0;
}
```

Reply:
Robert and I have both independently fixed this. I used stringstream and checked for the right flag, but Robert just did this with stoi.